doc-content/enterprise-only/project-data/test-scenario-preview-editor-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-scenario-preview-editor-con']
+= Using the Test Scenario (Preview) editor in {CENTRAL}
+
+{PRODUCT} ships with two editors for you to create and run test scenarios to validate your business rules and data. The difference between them is that the *Test Scenario* editor is supported by Red Hat for production use whereas *Test Scenario (Preview)* editor is a technology preview only and is not supported for production use. The sections covered in this chapter will help you understand on how to use the preview editor to create and run test scenarios.
```

Comment:
"...will help you understand on how to use ...."
"on" is not required. "...will help you understand  how to use ..."

Reply:
"on" removed.